src/messaging/ExchangeMgr.cpp

```diff
@@ -141,7 +141,11 @@ CHIP_ERROR ExchangeManager::UnregisterUnsolicitedMessageHandlerForType(Protocols
 
 void ExchangeManager::OnReceiveError(CHIP_ERROR error, const Transport::PeerAddress & source, SecureSessionMgr * msgLayer)
 {
-    ChipLogError(ExchangeManager, "Accept FAILED, err = %s", ErrorStr(error));
+    char srcAddressStr[Transport::PeerAddress::kMaxToStringSize];
```

Comment:
nit: might be worth guarding this code with `#if CHIP_ERROR_LOGGING` (not sure if it will be optimized out when logging is disabled).